internal/agent/command_config.go

```diff
@@ -0,0 +1,48 @@
+package agent
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/newrelic/newrelic-cli/internal/output"
+	"github.com/newrelic/newrelic-cli/internal/utils"
+)
+
+var (
+	encodeKey    string
+	textToEncode string
+)
+
+var cmdConfig = &cobra.Command{
+	Use:   "config",
+	Short: "Configuration utilities/helpers for New Relic agents",
+}
+
+var cmdConfigObfuscate = &cobra.Command{
+	Use:   "obfuscate",
+	Short: "Obfuscate a configuration value using a key",
+	Long: `Obfuscate a configuration value using a key.  Obfuscated value
+is placed in the Agent configuration or environment variable." 
+`,
+	Example: "newrelic agent config obfuscate --value <config_value> --key <obfuscation_key>",
+	Run: func(cmd *cobra.Command, args []string) {
+
+		result := ObfuscationResult{
+			ObfuscatedValue: obfuscateStringWithKey(textToEncode, encodeKey),
+		}
+
+		utils.LogIfFatal(output.Print(result))
+	},
+}
+
+func init() {
+
+	Command.AddCommand(cmdConfig)
+
+	cmdConfig.AddCommand(cmdConfigObfuscate)
+
+	cmdConfigObfuscate.Flags().StringVarP(&encodeKey, "key", "", "", "the key to use when obfuscating the clear-text value")
```

Comment:
The 3rd argument to `StringVarP` is a single letter to provide a short option for the argument (e.g. `--key` vs `-k`).  "k" and "v" might be good values for these two arguments.  If you're not interested in a short form options, you can just use `StringVar`.